datahub-web-react/src/app/shared/deleteUtils.ts

```diff
@@ -0,0 +1,56 @@
+import { useDeleteAssertionMutation } from '../../graphql/assertion.generated';
+import { useDeleteDomainMutation } from '../../graphql/domain.generated';
+import { useDeleteGlossaryEntityMutation } from '../../graphql/glossary.generated';
+import { useRemoveGroupMutation } from '../../graphql/group.generated';
+import { useDeleteTagMutation } from '../../graphql/tag.generated';
+import { useRemoveUserMutation } from '../../graphql/user.generated';
+import { EntityType } from '../../types.generated';
+
+/**
+ * Returns a relative redirect path which is used after an Entity has been deleted from it's profile page.
+ *
+ * @param type the entity type being deleted
+ */
+export const getEntityProfileDeleteRedirectPath = (type: EntityType) => {
+    switch (type) {
+        case EntityType.CorpGroup:
+        case EntityType.CorpUser:
+        case EntityType.Domain:
+        case EntityType.Tag:
+            // Return Home.
+            return '/';
+        case EntityType.GlossaryNode:
+        case EntityType.GlossaryTerm:
+            // Return to glossary page.
+            return '/glossary';
+        default:
+            return () => undefined;
```

Comment:
should we default to `/`? That seems like a reasonable fallback.

Reply:
Actually in some cases we may not want to do a redirect ...

Reply:
In case of assertion delete for example